doc/classes/XRFaceTracker.xml

```diff
@@ -0,0 +1,469 @@
+			Upper left lip part pushes in the cheek.
+		</constant>
+		<constant name="FT_MOUTH_UPPER_RIGHT" value="66" enum="BlendShapeEntry">
+			Moves upper lip right.
+		</constant>
+		<constant name="FT_MOUTH_UPPER_LEFT" value="67" enum="BlendShapeEntry">
+			Moves upper lip left.
+		</constant>
+		<constant name="FT_MOUTH_LOWER_RIGHT" value="68" enum="BlendShapeEntry">
+			Moves lower lip right.
+		</constant>
+		<constant name="FT_MOUTH_LOWER_LEFT" value="69" enum="BlendShapeEntry">
+			Moves lower lip left.
+		</constant>
+		<constant name="FT_MOUTH_CORNER_PULL_RIGHT" value="70" enum="BlendShapeEntry">
+			Right lip corner pulls diagonally up and out.
+		</constant>
+		<constant name="FT_MOUTH_CORNER_PULL_LEFT" value="71" enum="BlendShapeEntry">
+			Left lip corner pulls diagonally up and out.
+		</constant>
+		<constant name="FT_MOUTH_CORNER_SLANT_RIGHT" value="72" enum="BlendShapeEntry">
+			Right corner lip slants up.
+		</constant>
+		<constant name="FT_MOUTH_CORNER_SLANT_LEFT" value="73" enum="BlendShapeEntry">
+			Left corner lip slants up.
+		</constant>
+		<constant name="FT_MOUTH_FROWN_RIGHT" value="74" enum="BlendShapeEntry">
+			Right corner lip pulls down.
+		</constant>
+		<constant name="FT_MOUTH_FROWN_LEFT" value="75" enum="BlendShapeEntry">
+			Left corner lip pulls down.
+		</constant>
+		<constant name="FT_MOUTH_STRETCH_RIGHT" value="76" enum="BlendShapeEntry">
+			Mouth corner lip pulls out and down.
+		</constant>
+		<constant name="FT_MOUTH_STRETCH_LEFT" value="77" enum="BlendShapeEntry">
+			Mouth corner lip pulls out and down.
+		</constant>
+		<constant name="FT_MOUTH_DIMPLE_RIGHT" value="78" enum="BlendShapeEntry">
+			Right lip corner is pushed backwards.
+		</constant>
+		<constant name="FT_MOUTH_DIMPLE_LEFT" value="79" enum="BlendShapeEntry">
+			Left lip corner is pushed backwards.
+		</constant>
+		<constant name="FT_MOUTH_RAISER_UPPER" value="80" enum="BlendShapeEntry">
+			Raises and slightly pushes out the upper mouth.
+		</constant>
+		<constant name="FT_MOUTH_RAISER_LOWER" value="81" enum="BlendShapeEntry">
+			Raises and slightly pushes out the lower mouth.
+		</constant>
+		<constant name="FT_MOUTH_PRESS_RIGHT" value="82" enum="BlendShapeEntry">
+			Right side lips press and flatten together vertically.
+		</constant>
+		<constant name="FT_MOUTH_PRESS_LEFT" value="83" enum="BlendShapeEntry">
+			Left side lips press and flatten together vertically.
+		</constant>
+		<constant name="FT_MOUTH_TIGHTENER_RIGHT" value="84" enum="BlendShapeEntry">
+			Right side lips squeeze together horizontally.
+		</constant>
+		<constant name="FT_MOUTH_TIGHTENER_LEFT" value="85" enum="BlendShapeEntry">
+			Left side lips squeeze together horizontally.
+		</constant>
+		<constant name="FT_TONGUE_OUT" value="86" enum="BlendShapeEntry">
+			Tongue visibly sticks out of the mouth.
+		</constant>
+		<constant name="FT_TONGUE_UP" value="87" enum="BlendShapeEntry">
+			Tongue points upwards.
```

Comment:
Does not just apply here. Across the whole huge list there's an inconsistency. 
"Up", "Down", "Right"...? or "Upwards", "Downwards" and "To the right"...? I'm not sure which one to say, but it feels like it needs to be consistent across all of these.